setup.py

```diff
@@ -225,14 +303,15 @@ def finalize_options(self):
     ],
     cmdclass={"build_ext": CMakeBuild},
     zip_safe=False,
-    install_requires=["pycarl>=2.3.0"],
-    setup_requires=["pycarl>=2.3.0", "packaging"],  # required to check pybind version used for pycarl
+    install_requires=[],
+    setup_requires=["pytest-runner", "packaging"],
+    tests_require=["pytest", "nbval", "numpy"],
```

Comment:
```suggestion
    setup_requires=["packaging"],
```
see https://github.com/moves-rwth/stormpy/commit/82d707156a5597bec6fae4b732bcda87a86479df